### PR TITLE
feature/fix_vapor_pressure_deficit_check

### PR DESCRIPTION
**Pull request by dgergel**
This PR implements a check on vapor pressure deficit in the CESM driver (if VPD is < 0, it sets VPD to zero and updates VP) as is currently implemented in the image driver.

### CODE REVIEW

**Comment by jhamman** [comment]
See also https://github.com/UW-Hydro/VIC/pull/573.

**Comment by bartnijssen** [comment]
In a coupled model this should not happen, but good to have the check. However, I share Joe's concern about the potential for a (tiny) leak, so a `log_warn()` would be in order. The user can always turn those off by changing the `LOG_LEVEL`.

File: vic/drivers/cesm/src/vic_force.c [hunk 1]
```diff
@@ -205,6 +205,10 @@ vic_force(void)
         for (j = 0; j < NF; j++) {
             // vapor pressure deficit
             force[i].vpd[j] = svp(force[i].air_temp[j]) - force[i].vp[j];
+	    if (force[i].vpd[j] < 0) {
+		force[i].vpd[j] = 0;
+		force[i].vp[j] = svp(force[i].air_temp[j]);
+	    }
```
Comment (bartnijssen) [inline vic/drivers/cesm/src/vic_force.c]: Add `log_warn()` when this happens with info about the reset (display the value of `vpd` before the reset and say it was rest to 0).

**Comment by dgergel** [comment]
That makes sense - I've updated this to include a warning.

**Pull request merged**